tests/dummy/app/pods/field/controller.js

```diff
@@ -1,10 +1,10 @@
 import Ember from 'ember'
 const {
-  Controller
+  Controller, inject
```

Comment:
I agree, but that is another ESLint rule we should create (this stuff was automatically done by running `npm run eslint --- --fix`). Anything else we want done I want to create more rules for so we implement the fix once and get it everywhere for free.